.devcontainer/docker_taxi/devcontainer.json

```diff
@@ -0,0 +1,29 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/miniconda
+{
+	"name": "DSToolKit-DockerTaxi",
+	"context": "../../mlops/docker_taxi/environment",
+	"dockerFile": "../../mlops/docker_taxi/environment/Dockerfile",
```

Comment:
Help me understand the purpose and the end goal of the following Dockerfile environment implementation. 
1) Achieve an isolated environment to create isolated environments for each step of the CI/CD pipeline (build, test, deploy). Essentially, each step runs in the container, providing complete isolation. This will require having a common requirements.txt file and a single Dockerfile. 
2) Approach with the devcontainer provides an isolated environment for development, but not necessarily for each step of a CI/CD pipeline.

Reply:
The main goal is to allow developers to test their train, predict, etc. code and run unit tests locally. The base dev container is still available to submit pipeline jobs. As for CI/CD, the unit tests is the only step that currently is needing to be ran with this docker environment since they would likely use the same packages that are needed for the train, etc. code. Those packages could be pytorch or some other tool which could be quite large so having the multiple environments is ideal so a developer could choose the desired build.